charts/kube-state-metrics/templates/rolebinding.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if and (eq  .Values.rbac.create true) (eq .Values.rbac.useClusterRole false) -}}
+{{- range (split "," $.Values.namespace) }}
```

Comment:
```suggestion
{{- range (split "," $.Values.namespaces) }}
```

Reply:
> @KlavsKlavsen You have my sincere apologies for making you wait for this PR for so long. But I am afraid that we can only merge this PR once we go live with `kube-state-metrics v2.0` (correct me if I am mistaken @mrueg ).
> 
> The reason is that `.Values.namespace` is intended to be a single value field. We need to use the `--namespaces` field and that change has been made only in v2.0.
> 
> As a frequent user of helm charts myself, I feel it's in the best interest of open source users and maintainers to make the helm charts have the same cadence as that of the binary. In light of this, I think we should merge and publish this change only when kube-state-metrics v2 is released.

Let's keep this change on ksm-v1.9.x to be able to move forward and integrate it. I'm tracking v2 changes in 
https://github.com/kubernetes/kube-state-metrics/pull/1358

Reply:
In upstream 1.9 branch - the docs says: --namespace string Comma-separated list of namespaces to be enabled. Defaults to ""

So it has the exact same use for kube-state-metrics in 1.9.x as it has in 2.x - its just gotten an extra s for clarification.

So I'm not quite sure what you're referring to here, namespace also in 1.9.x supported multiple namespaces. ?